### DISCUSSION TITLE
An animation on my website works on localhost:3000 but not on my GitHub pages site.

### DISCUSSION THREAD

**Discussion by shriyanyamali** [community/community#136602] [Programming Help]
### Select Topic Area

Question

### Body

On my [GitHub pages site](https://shriyanyamali.github.io), on my [resume page](https://shriyanyamali.github.io/resume), as the user scrolls down, the circles at each experience page should get more complete. This works at localhost:3000 but after deploying to my GitHub pages website, the circle in the Volunteer section and the first 4 circles in the Experience section do not start completing until I reach the fifth circle. I am using Google Chrome, but it also does not work on Firefox. I have also tried using different screen sizes. The animation uses Framer Motion.

Link to my repo: [https://github.com/ShriyanYamali/shriyanyamali.github.io](https://github.com/ShriyanYamali/shriyanyamali.github.io)

The relevant files are src/pages/[resume.js](https://github.com/ShriyanYamali/shriyanyamali.github.io/blob/main/src/pages/resume.js), src/components/[LiIcon.js](https://github.com/ShriyanYamali/shriyanyamali.github.io/blob/main/src/components/LiIcon.js), src/components/[Education.js](https://github.com/ShriyanYamali/shriyanyamali.github.io/blob/main/src/components/Education.js), , src/components/[Experience.js](https://github.com/ShriyanYamali/shriyanyamali.github.io/blob/main/src/components/Experience.js), , and src/components/[Volunteering.js](https://github.com/ShriyanYamali/shriyanyamali.github.io/blob/main/src/components/Volunteering.js).

**Comment by LiteBrite82** [comment]
Im taking a look, but I'm not up to speed on Next.js or Framer Motion. 

I'm also going to move this to Programming Help (this could belong in either category), they are a little more active with this type of stuff.
